src/test/java/com/cgdpd/library/validation/ValidatorTest.java

```diff
@@ -84,35 +84,36 @@ void should_throw_exception_when_the_value_is_null() {
     }
 
     @Test
-    void should_throw_exception_when_date_value_is_after_now() {
-        //given
+    void should_throw_exception_when_year_value_is_after_now() {
+        // Given
         String paramName = "value";
-        Optional<LocalDate> value = Optional.of(LocalDate.now().plusDays(1));
+        Optional<Short> value = Optional.of((short) (LocalDate.now().getYear() + 1));
 
-        //when
+        // When
         Throwable thrownException = catchThrowable(
               () -> Validator.checkBeforeNow(paramName, value));
 
-        //then
+        // Then
         assertThat(thrownException)
               .isInstanceOf(ValidationException.class)
-              .hasMessage(String.format("Invalid %s. %s is after the current date", paramName,
+              .hasMessage(String.format("Invalid %s. %s is after the current year", paramName,
                     value.orElseThrow()));
     }
 
     @Test
-    void should_return_date_value_before_now() {
-        // given
+    void should_return_year_value_before_now() {
+        // Given
```

Comment:
Why the uppercase? let's leave it as before, no need to make our lives more difficult by uppercasing first letter everywhere